doc/radosgw/group.rst

```diff
@@ -0,0 +1,100 @@
+===============
+Groups
+===============
+
```

Comment:
@cbodley : I am really not sure which version this will go into, because the REST APIs are still not in place and I am still figuring things out, and Adam thinks that the REST APIs are a must.